sqlglot/dialects/druid.py

```diff
@@ -0,0 +1,105 @@
+from sqlglot import exp, generator, parser
+from sqlglot.dialects.dialect import Dialect
+from sqlglot.tokens import TokenType
+
+
+class TemporalFloor(exp.Func):
```

Comment:
we don't have dialect level expressions, please move these to expressions.py, you also need to add rendering in the generic dialect, we need to ensure all sql is convertible to generic.